app/components/card-expense.tsx

```diff
@@ -34,21 +34,24 @@ export function CardExpense({
         <ul>
           {expenses.map((expense) => (
             <li className="my-2" key={expense.transaction_code}>
-              <div className="flex justify-between">
-                <h3 className="text-lg font-medium">{expense.description}</h3>
-                <h3 className="text-lg font-medium">
-                  {new Intl.NumberFormat("id-ID", {
-                    style: "currency",
-                    currency: "IDR",
-                  }).format(expense.idr_amount.toNumber())}
-                </h3>
-              </div>
+              {/* TODO: use whatever we use for detail links. Eg: tx code, unique id, uuid, etc. */}
+              <a href={`/expenses/123`}>
```

Comment:
hmm, kenapa ga langsung pake `transaction_code`? atau dto nya belum final ya?

Reply:
that's the question, masih unclear posisinya, but I think I'll change it to `transaction_code` langsung

Reply:
Yeah I'd also prefer this to be a `/expenses/${expense.transaction_code}`